src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -39,6 +39,9 @@
     public static final String MESSAGE_SWITCH_SUCCESS = "Switching to %s tab";
     public static final String MESSAGE_SAME_SCREEN = "Already in %s tab";
 
-    /** Find message: First string refers to the number if item found */
-    public static final String MESSAGE_FIND_APPLICATION_SUCCESS = "%1$d applications listed!";
+    /** Find message: First string refers to the number if item found. second string refers to the item type. */
+    public static final String MESSAGE_FIND_SUCCESS = "%1$d %2$s listed!";
```

Comment:
Super minor point, but with regards to the phrasing, I suppose this might read a little awkwardly for more than 1 item found, such as "3 company listed!"? But it's just the phrasing so it's not a big deal and we can probably handle this in the future if we want!

Reply:
yeah i did thought about it, but it feels weird if we have to add in a condition for it LOL for whether to display the extra `s`. Hmm i think since we are dealing with a 1 vs Infinite amount, it feels like having the `s` should make sense right O:

Reply:
I think I will temporarily put this as `%1$d %2$s items found!`

Reply:
oh yea great idea!